internal/Square.py

```diff
@@ -29,5 +29,8 @@ def contains_friendly_piece(self, piece):
     def contains_opponent_piece(self, piece):
         return not self.is_free() and self.content.color != piece.color
 
+    def contains_opponent_pawn(self, piece):
+        return not self.is_free() and self.content.color != piece.color and self.content.is_pawn()
+
     def to_string(self):
-        return f'{util.FILE_NAMES[self.file]}{self.rank + 1}'
+        return f'{util.FILE_NAMES[self.file]}{8 - self.rank}'
```

Comment:
I think this constant's name is not really explicit, you may rename it by SPRITES_FILES_NAMES

Reply:
Here, the files are the "columns" of the board in chess vocabulary, which are letters, from 'a' to 'h'. If will rename it as "CHESSBOARD_FILE_NAMES", maybe